src/main/java/net/snowflake/client/jdbc/SnowflakeUtil.java

```diff
@@ -220,6 +220,11 @@ public static SnowflakeColumnMetadata extractColumnMetadata(
         extColTypeName = "BINARY";
         break;
 
+      case GEOMETRY:
+        colType = 12;
```

Comment:
12? Maybe it can be some Types reference?

Reply:
Yeah, 12 refers to Types.VARCHAR. I will update it to the reference variable.